src/mmw/js/src/app.js

```diff
@@ -131,9 +151,8 @@ function initializeShutterbug() {
                 'width': window.innerWidth
             });
 
-            var mapView = App.getMapView(),
-                activeBaseLayer = mapView.baseLayers[mapView.getActiveBaseLayerName()],
-                googleLayerVisible = !!activeBaseLayer._google;
+            var activeBaseLayer = App.getLayersModel().baseLayers.findWhere({ active: true }),
```

Comment:
To encapsulate implementation details (and since this same logic is repeated elsewhere) you might consider adding a helper method to `LayersModel` to retrieve the active base layer.